docs/docs/reference/contextual/multiversal-equality.md

```diff
@@ -112,24 +112,21 @@ The precise rules for equality checking are as follows.
 
 If the `strictEquality` feature is enabled then
 a comparison using `x == y` or `x != y` between values `x: T` and `y: U`
-is legal if
-
- 1. there is a given instance for `Eql[T, U]`, or
- 2. one of `T`, `U` is `Null`.
+is legal if there is a given instance for `Eql[T, U]`.
```

Comment:
Indeed, that is the current behaviour as sown bellow.

```scala
import scala.language.strictEquality
type X
type Y
object A {
  (??? : X) == (??? : Y)
  (??? : Null) == (??? : Y)
  (??? : X) == (??? : Null)
}
```
fails to compile with 

```scala
-- Error: Foo.scala:5:2 --------------------------------------------------------
5 |  (??? : X) == (??? : Y)
  |  ^^^^^^^^^^^^^^^^^^^^^
  |  Values of types X and Y cannot be compared with == or !=
-- Error: Foo.scala:6:2 --------------------------------------------------------
6 |  (??? : Null) == (??? : Y)
  |  ^^^^^^^^^^^^^^^^^^^^^^^^
  |  Values of types Null and Y cannot be compared with == or !=
-- Error: Foo.scala:7:2 --------------------------------------------------------
7 |  (??? : X) == (??? : Null)
  |  ^^^^^^^^^^^^^^^^^^^^^^^^
  |  Values of types X and Null cannot be compared with == or !=
```

But it is probably a bug in the implementation. @odersky WDYT?